modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -4340,6 +4370,18 @@ private static File _testNotExists(File dir, String fileName) {
 		return file;
 	}
 
+	private static void _testPropertyKeyExists(Path path, String key)
```

Comment:
The other `_testXXX` methods pass `File` and it'd be nice to use `File` here for consistency